quarkus/config-api/src/main/java/org/keycloak/config/Option.java

```diff
@@ -90,4 +91,14 @@ private static String getDescriptionByCategorySupportLevel(String description, O
 
         return description;
     }
+
+    public static String getDefaultValueString(Object value) {
```

Comment:
Why does it have to be static and compute it always for these options? Most of the time, we have the PropertyMapper present, so we can avoid doing this:
```
Option.getDefaultValueString(mapper.getDefaultValue().orElse(null))
````
and do this instead:
```
mapper.getDefaultValueString()
```
Right?

@shawkins So we could have one class method for things where it's needed and one instance method, WDYT?

Reply:
> @shawkins So we could have one class method for things where it's needed and one instance method, WDYT?

It's possible of course, this static method is for continuity with how the stringification happens currently and so that we don't end up with instance methods on both PropertyMapper and Option. I don't have a strong opinion though as an instance method was what I started with as well.